dask_cloudprovider/aws/ecs.py

```diff
@@ -434,24 +435,25 @@ def __init__(
         self._mem = mem
         self._gpu = gpu
         self._nthreads = nthreads
+        _command = [
+            "dask",
+            "cuda" if self._gpu else None,
```

Comment:
I'm apprehensive about having a `None` in this string. Have you tested that this works as expected?

Reply:
There's a None but it gets filtered out before we use it.

Reply:
I found the introduction and filtering of `None` hard to parse when reviewing this.

I think it would be more readable to have something like

```python
if self._gpu:
    _command = ["dask", "cuda", "worker"]
else:
    _command = ["dask", "worker"]
_command += [OTHERARGS...]
```